src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Roslyn/Services/MSBuildProjectService.cs

```diff
@@ -53,13 +53,12 @@ private void Initialize(bool refresh = false)
                 return;
             }
 
-            //try loading MSBuild project the faster way.
-            _project = new Project(_projectPath, null, null, new Build.Evaluation.ProjectCollection(), ProjectLoadSettings.IgnoreMissingImports);
-            //if fails, use the longer way using GLobalProjectCollection.
-            if (_project is null)
+            try
             {
-                _project = ProjectCollection.GlobalProjectCollection.LoadProject(_projectPath);
```

Comment:
This backup is no longer needed?

Reply:
yep this needs to be reworked. The backup wasn't particularly working so need a better way to go about this.